corehq/apps/app_manager/models.py

```diff
@@ -4404,15 +4425,18 @@ def odk_profile_display_url(self):
     def odk_media_profile_display_url(self):
         return self.short_odk_media_url or self.odk_media_profile_url
 
-    def get_odk_qr_code(self, with_media=False):
+    def get_odk_qr_code(self, with_media=False, profile=None):
```

Comment:
Should `profile` be called `build_profile` or am I misreading the meaning of that arg? (If there's a distinction, might be worth documenting that arg in the docstring